sql/src/test/java/io/crate/integrationtests/IngestRulesIntegrationTest.java

```diff
@@ -88,4 +89,19 @@ public void testDropMissingRuleIfExistsReturnsZeroAffectedRowsCount() {
         execute("drop ingest rule if exists somerule");
         assertThat(response.rowCount(), is(0L));
     }
+
+    @Test
+    public void testRenameTableTransfersIngestRules() {
+        execute("alter table t1 rename to t2");
+
+        execute("select rule_name from information_schema.ingestion_rules where target_table = 'doc.t2'");
+        assertThat(response.rowCount(), is(1L));
+        assertThat(response.rows()[0][0], is(INGEST_RULE_NAME));
```

Comment:
should we also assert that there is no ingestion rule in `information_schema.ingestion_rules` for `doc.t1` ?